test/terraform/validate.bats

```diff
@@ -1,9 +1,25 @@
 load 'lib'
 
+function setup() {
+  skip_unless_terraform
+  clean
+  export TF_CLI_ARGS_init="-input=false -backend=false"
+  terraform init
+}
+
+function teardown() {
+  clean
+  unset TF_CLI_ARGS_plan
+}
+
 @test "check if terraform code is valid" {
   skip_unless_terraform
-  run terraform validate -check-variables=false
-  log "$output"
-  [ $status -eq 0 ]
-  [ -z "$output" ]
+  if [[ "`terraform version | head -1`" =~ 0.12 ]]; then
```

Comment:
why the `.` is not escaped here in `=~ 0.12`, but escaped in https://github.com/cloudposse/test-harness/pull/7/files#diff-a301e3910214949a4fe851f846db12a0R50

Reply:
Good Catch

Reply:
```suggestion
  if [[ "`terraform version | head -1`" =~ 0\.12 ]]; then
```